src/sentry/utils/snuba.py

```diff
@@ -560,6 +560,9 @@ def snuba_query(params):
                     op="snuba", description=u"query {}".format(body)
                 ) as span:
                     span.set_tag("referrer", headers.get("referer", "<unknown>"))
+                    for extra_key in ["conditions", "groupby", "aggregations", "selected_columns"]:
+                        if extra_key in query_params:
+                            span.set_data(extra_key, query_params.get(extra_key))
```

Comment:
This is how you store dictionaries in spans !! Damn,  now that I know it works I will do that on the snuba side as well